Fatal Python error: initsite: Failed to import the site module
Traceback (most recent call last):
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/site.py", line 579, in <module>
    main()
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/site.py", line 566, in main
    known_paths = addsitepackages(known_paths)
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/site.py", line 349, in addsitepackages
    addsitedir(sitedir, known_paths)
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/site.py", line 207, in addsitedir
    addpackage(sitedir, name, known_paths)
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/site.py", line 168, in addpackage
    exec(line)
  File "<string>", line 1, in <module>
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/importlib/util.py", line 14, in <module>
    from contextlib import contextmanager
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/contextlib.py", line 5, in <module>
    from collections import deque
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/collections/__init__.py", line 1155, in <module>
    class UserString(_collections_abc.Sequence):
  File "/Users/caizhuo/opt/anaconda3/lib/python3.7/collections/__init__.py", line 1255, in UserString
    def isnumeric(self): return self.data.isnumeric()
KeyboardInterrupt
